src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx

```diff
@@ -395,31 +395,45 @@ export class SavedObjectsTable extends Component<SavedObjectsTableProps, SavedOb
     saveAs(blob, 'export.ndjson');
 
     const exportDetails = await extractExportDetails(blob);
-    this.showExportSuccessMessage(exportDetails);
+    this.showExportCompleteMessage(exportDetails);
     this.setState({ isShowingExportAllOptionsModal: false });
   };
 
-  showExportSuccessMessage = (exportDetails: SavedObjectsExportResultDetails | undefined) => {
+  showExportCompleteMessage = (exportDetails: SavedObjectsExportResultDetails | undefined) => {
     const { notifications } = this.props;
-    if (exportDetails && exportDetails.missingReferences.length > 0) {
-      notifications.toasts.addWarning({
-        title: i18n.translate(
-          'savedObjectsManagement.objectsTable.export.successWithMissingRefsNotification',
-          {
-            defaultMessage:
-              'Your file is downloading in the background. ' +
-              'Some related objects could not be found. ' +
-              'Please see the last line in the exported file for a list of missing objects.',
-          }
-        ),
-      });
-    } else {
-      notifications.toasts.addSuccess({
-        title: i18n.translate('savedObjectsManagement.objectsTable.export.successNotification', {
-          defaultMessage: 'Your file is downloading in the background',
-        }),
-      });
+    if (exportDetails) {
+      if (exportDetails.missingReferences.length > 0) {
+        return notifications.toasts.addWarning({
+          title: i18n.translate(
+            'savedObjectsManagement.objectsTable.export.successWithMissingRefsNotification',
+            {
+              defaultMessage:
+                'Your file is downloading in the background. ' +
+                'Some related objects could not be found. ' +
+                'Please see the last line in the exported file for a list of missing objects.',
+            }
+          ),
+        });
+      }
+      if (exportDetails.excludedObjects.length > 0) {
+        return notifications.toasts.addWarning({
```

Comment:
I wonder if we should use `notifications.toasts.addInfo` instead since this is going to be expected behavior in most cases, rather than an issue that users should be concerned about.

Reply:
Yea, I think that makes sense